native/packages/react-native-bpk-component-card/src/BpkCard.js

```diff
@@ -83,40 +78,34 @@ const styles = StyleSheet.create({
 
 const BpkCard = (props) => {
   const {
-    padded, children, focused, onPress, style, ...rest
+    padded,
+    children,
+    focused,
+    style: userStyle,
+    ...rest
   } = props;
 
-  const cardBaseStyle = [styles.card, styles.common];
-
-  if (padded) {
-    cardBaseStyle.push(styles.padded);
-  }
-
-  if (focused) {
-    cardBaseStyle.push(styles.focused);
-  }
+  const style = [styles.card];
+  if (padded) { style.push(styles.cardPadded); }
+  if (focused) { style.push(styles.cardFocused); }
+  if (userStyle) { style.push(userStyle); }
 
   return (
     <TouchableHighlight
       accessibilityComponentType="button"
-      onPress={onPress}
-      style={[cardBaseStyle, style]}
```

Comment:
Use *one* mechanism to merge styles. Either if statements above the JSX or rely on react natives ability to flatten arrays - not both.